addons/sys_core/fnc_coreInitPFH.sqf

```diff
@@ -0,0 +1,38 @@
+/*
+ * Author: ACRE2Team
+ * Informs, once all checks are passed, that all ACRE2 core components are properly loaded. It is executed
+ * as a per frame handler until all core checks are successful.
+ *
+ * Arguments:
+ * None
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [] call acre_sys_core_fnc_coreInitPFH
+ *
+ * Public: No
+ */
+#include "script_component.hpp"
+
+if (isNull player) exitWith { };
+acre_player = player;
+
+if (!ACRE_MAP_LOADED) exitWith { };
+if (!ACRE_DATA_SYNCED) exitWith { };
+if (GVAR(ts3id) == -1) exitWith { };
```

Comment:
`if (!ACRE_MAP_LOADED || {!ACRE_DATA_SYNCED} || {GVAR(ts3id) == 1}) exitWith {};`
Possibly not faster, but cleaner.